### PR TITLE
DOCS-8625-WSC-examples-edits-kt

### PR DESCRIPTION
**Pull request by kevintroller**
WSC 1.6.1 Examples doc

### CODE REVIEW

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
+= Web Service Consumer Connector Examples - Mule 4
+
+The following example shows how to transform a variable value into XML and pass that value to the Web Service Consumer Connector that subsequently consumes a web service.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
The following example shows how to transform a variable value into XML and pass that value to the Web Service Consumer Connector that subsequently consumes a web service:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.Web Service Consumer Connector Example Flow
+image::web-service-consumer-example1.png[Web Service Consumer Example Flow]
+
+== Web Service Consumer Example in Anypoint Studio
+To create the flow in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Core* > *Scheduler*.
+. Drag *Scheduler* component to the Studio canvas. +
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. Drag the *Scheduler* component to the Studio canvas. +
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+The *Scheduler* component initiates a flow when a time-based condition is met.
+. Set the *Display Name* field to `Scheduler`.
+. Set the *Scheduling Strategy* to `Fixed Frequency`, *Frequency* to `1000`, *Start delay* to `0` and *Time unit* to `MILLISECONDS (Default)`.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. Set *Scheduling Strategy* to `Fixed Frequency`, *Frequency* to `1000`, *Start delay* to `0` and *Time unit* to `MILLISECONDS (Default)`.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+In Studio, the *Set Variable* configuration looks like this:
+
+.Set Variable Configuration
+image::web-service-consumer-example3.png[Set Variable Configuration]
+
+[start=8]
+. Drag a *Transform Message* component onto the canvas, to the right of the *Set Variable* component. +
+The *Transform Message* component transforms the variable value into XML.
+. In the *Output* field of the *Transform Message* component configuration, set the DataWeave expression that:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. In the *Output* field of the *Transform Message* component configuration, set a DataWeave expression that:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+* Transforms the variable value output to `application/xml`.
+* Defines the name of the web service operation that the Web Service Consumer aims to invoke `attachOperation`.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]: I got a bit confused by capping web service consumer (did you mean the connector name?) and the use of "aims to", so...:
```suggestion
* Defines the name of the web service operation invoked by `attachOperation`.
```
?

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+}
+----
+
+In Studio, the *Transform Message* configuration looks like this:
+
+.Transform Message Configuration
+image::web-service-consumer-example4.png[Transform Message Configuration]
+
+[start=10]
+. Drag the Web Service Consumer *Consume* operation onto the canvas, to the right of the *Transform Message* component. +
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. Drag the *Consume* operation onto the canvas, to the right of the *Transform Message* component. +
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+. Set the *Display Name* field to `Consume`.
+. Set the *Operation* to `attachOperation` and the *Body* message to `payload`.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. Set *Operation* to `attachOperation` and the *Body* message to `payload`.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+. In the *Message Customization* section of the *Advanced* tab, enable the option *Force XML Prolog into body* to append the XML Prolog into the envelope's body XML content.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. In the *Message Customization* section of the *Advanced* tab, enable the option *Force XML Prolog into body* to append the XML prolog to the content of the envelope's body XML.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+.Consume Operation Configuration
+image::web-service-consumer-example5.png[Consume Operation Configuration]
+
+In Studio, the *XML Prolog* configuration looks like this:
+
+.Configure XML Prolog into Body
+image::web-service-consumer-xmlprolog.png[Configure XML Prolog into Body]
+
+[start=14]
+. On the *General* tab,  click the plus sign (+) next to the *Connector configuration* field to access the *Global Element Properties* configuration.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
. On the *General* tab, click the plus sign (+) next to the *Connector configuration* field to access the *Global Element Properties* configuration.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+. On the *Connection* section of the *General* tab, configure the following parameters:
+* *WSDL location* +
+The WSDL file URL remote or local.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
The WSDL file URL, remote or local.
```
OR
```suggestion
The remote or local WSDL file URL
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+* *Service* +
+The service name.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
The service name
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+* *Port* +
+The port name.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
The port name
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+* *Address* +
+The address of the web server to dispatch requests but if the *Service* and *Port* parameters don't auto-populate the web server address, you must manually provide the correct *Address*.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
The address of the web server to dispatch requests if the *Service* and *Port* parameters did not automatically provide this value. 
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+
+.Web Service Consumer Configuration
+image::web-service-consumer-example6.png[Web Service Consumer Configuration]
+
+[start=17]
+. Save your changes in Studio.
+. Deploy the application.
+
+== Web Service Consumer Example in XML
+In the *Configuration XML* tab in Studio, the XML looks as follows. If needed, change the values to reflect your environment:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]:
```suggestion
In the *Configuration XML* tab in Studio, the XML looks like this (if needed, change the values to reflect your environment):
```

Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc]: ...or however Diane words it. I think it's a little different from what you had originally.

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**